src/networking/APNSNetworkService.php

```diff
@@ -73,7 +70,10 @@ public function enqueueRequest( string $url, array $headers, string $body ): voi
 		curl_setopt( $ch, CURLOPT_POSTFIELDS, $body );
 		curl_setopt( $ch, CURLOPT_VERBOSE, $this->debug );
 		curl_setopt( $ch, CURLOPT_PORT, $this->port );
-		curl_setopt( $ch, CURLOPT_CAINFO, $this->certificate_bundle_path );
+
+		if( ! is_null( $this->certificate_bundle_path ) ) {
```

Comment:
```suggestion
		if ( ! is_null( $this->certificate_bundle_path ) ) {
```

Gotta thanks CI and running `phpcbf` locally, because I did not notice it.

Reply:
By the way, I made this pre-commit hook to help me with the feedback cycle here. It caught quite a few things for me before I committed them.

```bash
#!/bin/bash

set -e

# Unit tests with no coverage so they're faster (even if just a tiny bit)
./vendor/bin/phpunit --no-coverage --color=always

# Run PHPCBF for each of the staged files.
files=''
while read file
do
  files="${files} ./${file}"
done < <(git diff --cached --name-only --diff-filter=AM)

if [[ $files = '' ]]; then
  # This can happen when amending a commit to fix a typo in the message
  echo "Skipping PHPCBF because there is no staged file and it takes too long to run."
else
  ./vendor/bin/phpcbf ${files}
fi

# Run Psalm on the whole codebase
./vendor/bin/psalm
```

Reply:
This is great! It should be part of the project!! 😄